.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "singleQuote": true,
+  "jsxSingleQuote": false,
```

Comment:
I believe we should be able to use single quotes in jsx. They can tend to look a bit cleaner.

Reply:
Oh yeah! I forgot to set it true. will update it